contracts/SmartLoan.sol

```diff
@@ -37,7 +31,7 @@ error LoanInsolvent();
 error PostSelloutLtvTooLow();
 
 /// This operation would not result in bringing the loan back to a solvent state
-error PostSelloutLoanInsolvent();
+error PostCloseLoanInsolvent();
```

Comment:
maybe `LoanInsolventAfterClose` will be more clear?

Reply:
How about `LoanInsolventAfterClosure`?

Reply:
sounds good :)